tests/test_settings.py

```diff
@@ -46,18 +46,16 @@ def test_toml_override(self, mock_getcwd):
         # create temporary toml config file
         dummy_conf_path = os.path.join(temp_dir, "pyproject.toml")
         os.makedirs(os.path.dirname(dummy_conf_path), exist_ok=True)
-        toml_conf_content = {
-            "tool": {
-                "foo": {"bar": "baz"},
-                "semantic_release": {
-                    "upload_to_pypi": False,
-                    "version_source": "tag",
-                    "foo": "bar",
-                },
-            },
-        }
+        toml_conf_content = '''
+[tool.foo]
+bar = "baz"
+[tool.semantic_release]
+upload_to_pypi = false
+version_source = "tag"
+foo = "bar"
+'''
         with open(dummy_conf_path, "w") as dummy_conf_file:
-            toml.dump(toml_conf_content, dummy_conf_file)
+            dummy_conf_file.write(tomlkit.dumps(tomlkit.loads(toml_conf_content)))
```

Comment:
Would it be possible to write the file directly?
```py
dummy_conf_file.write(toml_conf_content)
```

Reply:
Yes, it's a way to validate the toml configuration before writing the file, but I'll update this.